modules/tinymce/src/themes/silver/main/ts/ui/core/complex/StyleSelect.ts

```diff
@@ -22,7 +23,7 @@ const getSpec = (editor: Editor, dataset: SelectDataset): SelectSpec => {
   const getPreviewFor = (format: string) => () => {
     const fmt = editor.formatter.get(format);
     return fmt !== undefined ? Optional.some({
-      tag: fmt.length > 0 ? fmt[0].inline || fmt[0].block || 'div' : 'div',
+      tag: fmt.length > 0 ? (fmt[0] as InlineFormat).inline || (fmt[0] as BlockFormat).block || 'div' : 'div',
```

Comment:
Type guards?

Reply:
That would change the code/functionality, so out of scope.